src/App.jsx

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+
+import React, { Component } from 'react';
+import Button from './components/Button';
+import './App.css';
+
+import storage from './storage/storage';
+import {
+  INFINITY_ERROR_TEXT,
+  CALCULATOR_DATA_KEY,
+  OPERATOR,
+  CALCULATOR_NUMBER_LIST,
+  CALCULATOR_OPERATOR_LIST,
+} from './constants';
+import {
+  validateOperatorIsDuplicated,
+  isArithmeticOperator,
+  toFixedValue,
+} from './utils';
+
+class App extends Component {
+  #expressionRef;
+
+  constructor() {
+    super();
+
+    this.#expressionRef = React.createRef();
+    this.state = {
+      firstOperand: 0,
+      secondOperand: 0,
+      operator: '',
+      calculationResult: 0,
+    };
+
+    window.addEventListener('beforeunload', this.#handleBeforeUnload);
+    window.addEventListener('unload', this.#handleUnload);
+  }
+
+  componentDidMount() {
+    if (storage.get(CALCULATOR_DATA_KEY)) {
+      const {
+        firstOperand,
+        secondOperand,
+        operator,
+        calculationResult,
+        lastExpression,
+      } = storage.get(CALCULATOR_DATA_KEY);
+
+      this.setState({
+        firstOperand,
+        secondOperand,
+        operator,
+        calculationResult,
+      });
+
+      this.#expressionRef.current.textContent = lastExpression ?? 0;
+    }
+  }
+
+  #handleBeforeUnload = (e) => {
+    e.preventDefault();
+    e.returnValue = '';
+  };
+
+  #handleUnload = () => {
+    const lastExpression = Number(this.#expressionRef.current.textContent);
+    storage.set(CALCULATOR_DATA_KEY, { ...this.state, lastExpression });
+  };
+
+  #handleDigitClick = (e) => {
+    const expression = this.#expressionRef.current.textContent;
+    const digit = e.target.textContent;
+
+    if (isArithmeticOperator(expression) || expression === '0') {
+      this.#expressionRef.current.textContent = digit;
+      return;
+    }
+
+    this.#expressionRef.current.textContent += digit;
+  };
+
+  #handleModifierClick = (e) => {
```

Comment:
```suggestion
  #handleModifierClick = (_) => {
```
사용하지 않는 인자는 _(언더스코어)를 사용합시다~
중복해서 사용하지 않는 인자가 있다면 언더스코어의 개수를 늘려가면 되요!

e.g
```javascript
#handleArguments(_, __, valid) {
   // ...
}
```